QUEST.tsv

```diff
@@ -628,8 +628,8 @@ QUEST_20150317_000627	When the great swordsman, Ruklys, completed the fortress,
 QUEST_20150317_000628	Their power grew to that of a city-state over time,{nl}and it is suspected that Ruklys' ambition grew as well.
 QUEST_20150317_000629	Ruklys' influence grew so much that he eventually revolted against King Kadumel.
 QUEST_20150317_000630	It was a huge rebellion. The entire city took part. {nl}That's why some record it as a civil war.
-QUEST_20150317_000631	The rebellion initially favored Ruklys, but tides turned {nl}when the great Premier Eminent and marksman Lydia Schaffen joined the war.
-QUEST_20150317_000632	It all ended two years after, when Ruklys died at the hands of Lydia Schaffen.
+QUEST_20150317_000631	The rebellion initially favored Ruklys, but, the tides turned {nl}when the great Premier Eminent and marksman Lydia Schaffen joined the war.
```

Comment:
thank you! so, does a maintainer of the ToS official english translation repository need to do the merge or is that something i initiate on my end?